openreview/conference/builder.py

```diff
@@ -1754,24 +1754,27 @@ def is_release_authors(is_note_accepted):
         self.expire_recruitment_invitations()
 
     def send_decision_notifications(self, decision_options, messages):
-        decision_notes = self.client.get_all_notes(
-            invitation=self.get_invitation_id(self.decision_stage.name, '.*'),
-        )
-        paper_notes = {n.forum: n for n in self.get_submissions()}
+        paper_notes = self.get_submissions(details='directReplies')
 
         def send_notification(note):
-            paper_note = paper_notes[note.forum]
-            message = messages[note.content['decision']]
+            decision_note = None
+            for reply in note.details['directReplies']:
+                if reply['invitation'].endswith(self.decision_stage.name):
```

Comment:
```suggestion
                if reply['invitation'].endswith('/-/' + self.decision_stage.name):
```

in case there are other invitations that end with the same name